### PR TITLE
Updated to store learning path in DB as well

### PR DESCRIPTION
**Pull request by MarcNor**
Updated code to store the generated Learning Path in the DB as well. For this, updated the learning_path model as well according to needed data in __init__.py and adjusted tests as well.

### CODE REVIEW

File: repositories/repository.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import abc
-from domain.tutoringModel import learning_path
-from repositories.orm import LearningPathOrm
+from operator import mod
```
Comment (yusme) [inline repositories/repository.py:2]: This is not used anywhere

File: repositories/orm.py [hunk 2]
```diff
@@ -1,22 +1,19 @@
-from importlib.metadata import metadata
-from sqlalchemy import MetaData, Column, Integer, String
+from sqlalchemy import MetaData, Column, Integer, String, Table
 from sqlalchemy.orm import mapper, relationship
```
Comment (yusme) [inline repositories/orm.py:2]: relationship is not used anywhere

File: service_layer/services.py [hunk 3]
```diff
@@ -1,12 +1,21 @@
+from multiprocessing.sharedctypes import Value
```
Comment (yusme) [inline service_layer/services.py:1]: Value is not used anywhere

File: repositories/repository.py (hunk 1 above)
Comment (MarcNor) [reply repositories/repository.py:2]: removed

File: repositories/orm.py (hunk 2 above)
Comment (MarcNor) [reply repositories/orm.py:2]: removed

File: service_layer/services.py (hunk 3 above)
Comment (MarcNor) [reply service_layer/services.py:1]: Not included

**Review by yusme — APPROVED** [review]

**Pull request merged**